Makefile

```diff
@@ -46,15 +46,7 @@ clean:
 	git clean -ff -d -x --exclude="$(ROOTDIR)/tests/externaldata/*" --exclude="$(ROOTDIR)/tests/data/*" --exclude="$(ROOTDIR)/conda/"
 
 compile_notebooks:
-	for f in $(shell find docs/tutorials -type f -name '*.md' -print) ; do \
-		DIR=`dirname $$f` ; \
-		BASENAME=`basename $$f` ; \
-		echo $$DIR $$BASENAME ; \
-		cd $$DIR ; \
-		python $(MD2IPYNB) $$BASENAME ; \
-		if [ $$? -ne 0 ] ; then echo "the following notebook raised an error:" $$BASENAME ; exit 1 ; fi ; \
-		cd - ; \
-	done;
+	python $(MD2IPYNB) "docs/tutorials/**/*.md"
```

Comment:
Is this equivalent of the deleted code? If so, nice!

Reply:
I hope so!